google/cloud/storage/benchmarks/aggregate_throughput_benchmark.cc

```diff
@@ -88,11 +88,6 @@ using Counters = std::map<std::string, std::int64_t>;
 struct DownloadDetail {
   int iteration;
   int thread_count;
```

Comment:
nit: `thread_count` can also be removed

(sorry, I misquoted this line in my original comment)

Reply:
Done.